x-pack/plugins/security_solution/public/exceptions/pages/shared_lists/index.tsx

```diff
@@ -384,10 +384,18 @@ export const SharedLists = React.memo(() => {
         <EuiFlexItem>
           <EuiPageHeader
             pageTitle={i18n.ALL_EXCEPTIONS}
-            description={timelines.getLastUpdated({
-              showUpdating: loading,
-              updatedAt: lastUpdated,
-            })}
+            description={
+              <>
+                <div>{'To view rule specific exceptions navigate to the rules details page.'}</div>
```

Comment:
Change to: `To view rule specific exceptions navigate to that rule's details page.`

Reply:
fixed here: dd7a59da6b0